### PR TITLE
Doc/swift package manager

### PR DESCRIPTION
**Pull request by nxtstep**
Forgot to update the README 😇

### CODE REVIEW

**Comment by nxtstep** [comment]
@robrix can you pls merge this one? Cheers 🙏

**Comment by robrix** [comment]
I’m no longer involved in Swift, but cc @guidomb, @nikita-leonov, @robb.

File: README.md [hunk 1]
```diff
@@ -165,3 +165,18 @@ Full API documentation is in the source.
 2. Drag `Prelude.xcodeproj` into your project or workspace.
 3. Link your target against `Prelude.framework`.
 4. Application targets should ensure that the framework gets copied into their application bundle. (Framework targets should instead require the application linking them to include Prelude.)
+
+Or use the Swift package manager and add this to you `Package.swift` file:
```
Comment (robrix) [inline README.md]: Typo: “add this to _your_” etc

**Comment by robb** [comment]
I think this looks good.

**Pull request merged**